.travis.yml

```diff
@@ -6,9 +6,16 @@ env:
     - CI_TARGET=vim MAKTABA_VERSION=1.14.0
     - CI_TARGET=vim MAKTABA_VERSION=master
     - CI_TARGET=neovim MAKTABA_VERSION=master
+    - CI_TARGET=vim MAKTABA_VERSION=master COVERAGEPY="coverage>4,<5"
 before_script:
   - sudo apt-get update
-  - sudo apt-get install python3-dev python3-coverage
+  - sudo apt-get install python3-dev
+  - if [ -z "$COVERAGEPY" ]; then
+      sudo apt-get install python-coverage3;
```

Comment:
At least some of the failures are from a typo here: Should be python3-coverage instead of python-coverage3.

Reply:
Thanks, fixed.